examples/linear_model/plot_lasso_coordinate_descent_path.py

```diff
@@ -60,10 +64,11 @@
 
 plt.figure(2)
 ax = plt.gca()
-ax.set_color_cycle(2 * ['b', 'r', 'g', 'c', 'k'])
-l1 = plt.plot(-np.log10(alphas_lasso), coefs_lasso.T)
-l2 = plt.plot(-np.log10(alphas_positive_lasso), coefs_positive_lasso.T,
-              linestyle='--')
+for coef in coefs_lasso:
+    l1 = plt.plot(-np.log10(alphas_lasso), coef, c=next(colors))
```

Comment:
This works because the list of colors is the same length as ``coefs_lasso``, right? That seems pretty implicit. cant we zip the coefs_lasso and coefs_enet and therefore make sure they have the same color?